runtime/service.go

```diff
@@ -514,14 +522,22 @@ func (s *service) StopVM(requestCtx context.Context, request *proto.StopVMReques
 		return nil, err
 	}
 
-	// The graceful shutdown logic, including stopping the VM, is centralized in Shutdown. We set "Now" to true
-	// to ensure that the service will actually shutdown even if there are still containers being managed.
-	_, err = s.Shutdown(requestCtx, &taskAPI.ShutdownRequest{Now: true})
-	if err != nil {
-		return nil, err
-	}
+	shutdownCh := make(chan error)
+	go func() {
+		defer close(shutdownCh)
+		_, err = s.Shutdown(requestCtx, &taskAPI.ShutdownRequest{Now: true})
```

Comment:
just noticed this, let's use `_, err := s.Shutdown(...)` (`:=` instead of `=`), just to avoid the confusion that can result from overwriting a named return val from within a separate goroutine.